engine/src/main/java/org/hibernate/validator/internal/engine/validationcontext/AbstractValidationContext.java

```diff
@@ -294,13 +295,15 @@ private String interpolate(
 			Object validatedValue,
 			ConstraintDescriptor<?> descriptor,
 			Map<String, Object> messageParameters,
-			Map<String, Object> expressionVariables) {
+			Map<String, Object> expressionVariables,
+			Path path) {
```

Comment:
Let's add the path after the descriptor. It makes more sense.

Reply:
fixed